src/imitation/scripts/config/train_adversarial.py

```diff
@@ -131,6 +131,7 @@ def cartpole():
 
 @train_adversarial_ex.named_config
 def seals_cartpole():
+    total_timesteps = 1.4e6
```

Comment:
Love the precision :) `1.5e6` would be too much, of course.

Reply:
Gotta speed up the iteration cycle :0